packages/@aws-cdk-testing/cli-integ/lib/shell.ts

```diff
@@ -21,9 +21,15 @@ export async function shell(command: string[], options: ShellOptions = {}): Prom
     }
   };
 
-  // Always output the command
-  writeToOutputs(`💻 ${command.join(' ')}\n`);
   const show = options.show ?? 'always';
+  const verbose = Boolean(process.env.VERBOSE);
```

Comment:
Oh wait maybe this is an intentional diff? Just noticed you [introduced](https://github.com/aws/aws-cdk/issues/33140) this a while go...

Reply:
Yes intentional